gui/src/components/Link.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import * as router from 'svelte-spa-router';
+
+  export let href: string;
+</script>
+
+<a
```

Comment:
Looks good. Might need an option to `display: block` as we use this broadly. Not sure yet.